openvmm/hvlite_entry/src/cli_args.rs

```diff
@@ -260,6 +260,10 @@ flags:
     #[structopt(long, value_name = "SERIAL")]
     pub vmbus_com2_serial: Option<SerialConfigCli>,
 
+    /// debugcon binding (port:serial, where port is a u16, and serial is (console | stderr | listen=\<path\> | term[=\<terminal emulator\>] | none))
```

Comment:
Note that I have an open PR that changes this text for the other serial ports.

Maybe we should figure out how to share this text, somehow.

Reply:
(Side note, the backslash part of `\<` doesn't seem to be right? But it's everywhere? Was it once right?)

Reply:
I believe we can do some shenanigans with `#[doc = macro_returning_a_literal!()]` here.

wrt. the `\<`, I believe this was fallout from steven fixing a doc-related clippy lint, related to rustdoc interpretting the brackets as a link

Reply:
(Oh, maybe rustdoc interprets <stuff> as inline HTML as part of markdown, but clap gleefully ignores it? Is that right?)

Reply:
(Yep, that's right. Hrm. Probably there's a better way to handle some of this.)

Reply:
Given that we have other instances of this sort of doc-duping in the CLI, lets do this on a separate PR.

Want to open an issue tracking this?

Reply:
#286